build/Microsoft.DotNet.Cli.Publish.targets

```diff
@@ -97,11 +97,13 @@
       AccountKey="$(CloudDropAccessToken)"
       AccountName="$(CloudDropAccountName)"
       ContainerName="$(ContainerName)"
-      Items="@(CliVersionBadgeToUpload)" />
+      Items="@(CliVersionBadgeToUpload)"
+      Overwrite="$(OverwriteOnPublish)" />
 
     <SetBlobPropertiesBasedOnFileType
       AccountKey="$(CloudDropAccessToken)"
       AccountName="$(CloudDropAccountName)"
+      ContainerName="$(ContainerName)"
```

Comment:
Because SetBlobPropertiesBasedOnFileType is hard-coded to "dotnet" without the changes in this PR, this task fails if ContainerName is set to something other than "dotnet", because the "dotnet" container won't exist or won't have the expected CLI build artifacts.